beacon_node/execution_layer/src/engine_api/http.rs

```diff
@@ -519,10 +556,47 @@ pub mod deposit_methods {
     }
 }
 
+#[derive(Clone, Debug)]
+pub struct CapabilitiesCacheEntry {
+    engine_capabilities: EngineCapabilities,
+    fetch_time: SystemTime,
+}
+
+impl CapabilitiesCacheEntry {
+    pub fn new(engine_capabilities: EngineCapabilities) -> Self {
+        Self {
+            engine_capabilities,
+            fetch_time: SystemTime::now(),
+        }
+    }
+
+    pub fn engine_capabilities(&self) -> &EngineCapabilities {
+        &self.engine_capabilities
+    }
```

Comment:
```suggestion
    pub fn engine_capabilities(&self) -> EngineCapabilities {
        self.engine_capabilities
    }
```

Super minor nit, but it makes a future suggestion slightly cleaner. `EngineCapabilities` is tiny and `Copy`, currently a reference/pointer is likely larger than the actual struct.